apps/frontend/src/views/Login.vue

```diff
@@ -4,61 +4,126 @@
       <v-container class="fill-height" fluid>
         <v-row align="center" justify="center">
           <v-col cols="12" sm="8" md="4">
-            <v-card class="elevation-12">
+            <v-card class="elevation-12 rounded-b-0">
               <v-toolbar color="primary" dark flat>
                 <v-toolbar-title id="login_form_title">
                   Login to Heimdall Server
                 </v-toolbar-title>
                 <v-spacer />
               </v-toolbar>
-              <v-card-text>
-                <v-form id="login_form" ref="form" name="login_form">
-                  <v-text-field
-                    id="email_field"
-                    v-model="email"
-                    :error-messages="emailErrors($v.email)"
-                    name="email"
-                    label="Email"
-                    prepend-icon="mdi-account"
-                    type="text"
-                    required
-                    @keyup.enter="$refs.password.focus"
-                    @blur="$v.email.$touch()"
-                  />
-                  <v-text-field
-                    id="password_field"
-                    ref="password"
-                    v-model="password"
-                    :error-messages="
-                      requiredFieldError($v.password, 'Password')
-                    "
-                    type="password"
-                    name="password"
-                    label="Password"
-                    prepend-icon="mdi-lock"
-                    @keyup.enter="login"
-                    @blur="$v.password.$touch()"
-                  />
-                  <v-btn
-                    id="login_button"
-                    depressed
-                    large
-                    color="primary"
-                    :disabled="$v.$invalid"
-                    @click="login"
-                  >
-                    Login
-                  </v-btn>
-                </v-form>
-              </v-card-text>
-              <v-card-actions>
-                <v-spacer />
-                <div class="my-2">
-                  <v-btn id="sign_up_button" depressed small @click="signup">
-                    Sign Up
-                  </v-btn>
-                </div>
-              </v-card-actions>
+              <v-tabs
+                active
+                :value="active_tab"
+                background-color="primary darken-1"
+                color="primary-visible"
+                show-arrows
+              >
+                <v-tab id="select-tab-standard-login" href="#login-standard"
+                  >Heimdall Login</v-tab
+                >
+                <v-tab id="select-tab-ldap-login" href="#login-ldap"
+                  >Organization Login</v-tab
+                >
```

Comment:
The actual code for these 2 pieces should be in their own component. There should be a LdapLogin component and a DatabaseLogin component.